dev/ci/ci-metacoq.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download metacoq
+
+( cd "${CI_BUILD_DIR}/metacoq" && make && make test-suite && make translations && make install)
```

Comment:
Why not make everything in one go?

Reply:
This is a multi-make project, I didn't try but suspect it would not respect the order